src/main/resources/assets/mmorpg/lang/ru_ru.json

```
@@ -0,0 +1,2718 @@
+	"mmorpg.stat_desc.cooking_medium_drop_bonus": "Добыча профессии делится на категории, эта характеристика увеличивает шанс дропа одной из них.",
+	"mmorpg.stat_desc.cooking_misc_drop_bonus": "Добыча профессии делится на категории, эта характеристика увеличивает шанс дропа одной из них.",
+	"mmorpg.stat_desc.cooking_triple_drop": "Шанс профессии утроить дроп, если это произойдет.",
+	"mmorpg.stat_desc.critical_damage": "Если критический, умножается на x.",
+	"mmorpg.stat_desc.critical_hit": "Шанс умножить урон атаки на критический урон.",
+	"mmorpg.stat_desc.crossbow_damage": "Увеличивает урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.curse_cdr": "Уменьшает время перезарядки заклинаний с указанным тегом.",
+	"mmorpg.stat_desc.damage_cdr": "Уменьшает время перезарядки заклинаний с указанным тегом.",
+	"mmorpg.stat_desc.damage_reflected": "Наносит % урона, который вы получаете, врагам, нападающим на вас.",
+	"mmorpg.stat_desc.damage_shield": "Уменьшает количество урона от каждой атаки.",
+	"mmorpg.stat_desc.dex_dmg": "Магический урон - это заклинания мага, например, огненный шар.",
+	"mmorpg.stat_desc.dex_dmg_received": "Магический урон - это заклинания мага, например, огненный шар.",
+	"mmorpg.stat_desc.dexterity": "Определяет ваше общее: Рейтинг уклонения, Регенерацию энергии, Энергию.",
+	"mmorpg.stat_desc.dmg_taken_to_mana": "% урона преобразуется в ману.",
+	"mmorpg.stat_desc.dmg_to_living": "Живые существа не являются нежитью.",
+	"mmorpg.stat_desc.dmg_when_low_hp": "Низкое здоровье составляет 30% или менее.",
+	"mmorpg.stat_desc.dmg_when_target_is_low_hp": "Низкое здоровье цели составляет 30% или менее.",
+	"mmorpg.stat_desc.dmg_when_target_near_full_hp": "70% здоровья и выше.",
+	"mmorpg.stat_desc.dodge": "Шанс игнорировать урон от атаки.",
+	"mmorpg.stat_desc.dot_dmg": "Увеличивает урон от эффектов, наносящих урон с течением времени, например, ожог.",
+	"mmorpg.stat_desc.dot_lifesteal": "Восстанавливает % урона от эффектов, наносящих урон с течением времени, в виде здоровья.",
+	"mmorpg.stat_desc.ele_atk_dmg": "Добавляет x элементарного урона при ударе оружием.",
+	"mmorpg.stat_desc.ele_axe_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_bow_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_crossbow_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_pene": "Игнорирует столько сопротивления, это работает как вычитание.",
+	"mmorpg.stat_desc.ele_resist": "Поглощает X процентов урона этого элемента.",
+	"mmorpg.stat_desc.ele_staff_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_sword_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
+	"mmorpg.stat_desc.ele_trident_damage": "Увеличивает элементарный урон, наносимый этим оружием.",
```

```suggestion
	"mmorpg.stat_desc.ele_atk_dmg": "Добавляет x урона от стихий при ударе оружием.",
	"mmorpg.stat_desc.ele_axe_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_bow_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_crossbow_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_pene": "Игнорирует столько сопротивления, это работает как вычитание.",
	"mmorpg.stat_desc.ele_resist": "Поглощает X процентов урона этого элемента.",
	"mmorpg.stat_desc.ele_staff_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_sword_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
	"mmorpg.stat_desc.ele_trident_damage": "Увеличивает урон от стихий, наносимый этим оружием.",
```
урон стихиями, урон стихий, стихийный урон

---

Исправлено